docs/pages/guides/editing-richtext.mdx

```diff
@@ -0,0 +1,118 @@
+
+Sometimes however, you need to be more flexible. Think of long social media posts, note apps or document editors. Ideally you need to allow different text styles, lists, headings, embedded images and more. This is called a rich text editor, and it’s a difficult problem to solve everywhere, including in React Native.
+
+There’s currently no obvious, default solution for that in the React Native ecosystem, but we can explore some options and promising approaches, each with their own tradeoffs.
+
+# Rendering rich text
+
+If you don’t need to actually edit rich text, just display it, there are a lot of great options:
+
+- If you have markdown content, you can use a markdown renderer like [react-native-markdown-renderer](https://www.npmjs.com/package/react-native-markdown-renderer) or other.
+
+- If you have HTML content, you can use [@expo/html-elements](https://www.npmjs.com/package/@expo/html-elements) or simply a webview ([react-native-webview](https://docs.expo.dev/versions/latest/sdk/webview/)).
+
+- If you have a custom format and want more control, you can take advantage of nesting `<Text>` components to render styles and layouts.
+
+  ```jsx
+  <TextInput>
+    <Text>
+      <Text style={{ fontWeight: 900 }}>Some bold text</Text>Some regular text
+    </Text>
+  </TextInput>
+  ```
+
+- You can also use [Expo Modules](/modules/overview/) to write a custom renderer component with native platform primitives such as `AttributedString` on iOS and 3rd party libraries such as [Markwon](https://github.com/noties/Markwon) on Android.
+
+# Editing rich text
+
+There are a few possible approches to get rich text rendering to work, but all have different limitations.
+
+## Webview-based editors
+
+While most react-native UI components wrap native platform primitives and are fast, performant and “native feeling” as a result, the webview-based rich text editors use a different approach.
+
+They wrap an existing rich text editor built for web with JavaScript inside a [react-native-webview](https://docs.expo.dev/versions/latest/sdk/webview/). It works on all platforms (Android, iOS, Web) and can take advantage of popular rich text editors available for the Web platform, but has a performance and UX penalty.
+
+You will not be able to use native UI components inside the editor. Any implementation of features like mentions or image embedding will duplicate features and require significant effort to implement.
+
+### Existing webview-based react-native packages
+
+There are a couple of existing react-native packages to allow rich-text editing. These are the easiest option to get started if you just need a basic rich text editor with limited configurability and don’t have strict performance or UX requirements.
+
+Here are some options:
+
+- [react-native-rich-editor](https://github.com/wxik/react-native-rich-editor)
+- [react-native-cn-quill](https://github.com/imnapo/react-native-cn-quill)
```

Comment:
```suggestion
- [`react-native-rich-editor`](https://github.com/wxik/react-native-rich-editor)
- [`react-native-cn-quill`](https://github.com/imnapo/react-native-cn-quill)
```

All library names when written in package format use inline syntax for consistency.